src/os/bluestore/NVMEDevice.cc

```diff
@@ -504,6 +503,8 @@ int NVMEManager::try_get(const string &sn_tag, SharedDriverData **driver)
   ProbeContext ctx = {sn_tag, this, nullptr};
   r = spdk_nvme_probe(&ctx, probe_cb, attach_cb);
   if (r || !ctx.driver) {
+    if (!ctx.driver)
+      r = -ENOENT;
```

Comment:
I don't think this is right. all errors will result in ctx.driver == nullptr, so we can't set r = -ENOENT indicate the specified error.